src/Exception/InvalidStopwatchEvent.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Exception InvalidStopwatchEvent.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Exception;
+
+use InvalidArgumentException;
+
+/**
+ * Exception thrown when an invalid stopwatch name was requested.
+ */
+final class InvalidStopwatchEvent
+	extends InvalidArgumentException
+	implements AmpWpException {
+
+	/**
+	 * Create a new instance of the exception for a stopwatch event name that is
+	 * not recognized but requested to be stopped.
+	 *
+	 * @param string $name Name of the event that was requested to be stopped.
+	 *
+	 * @return self
+	 */
+	public static function from_name_to_stop( $name ) {
+		$message = \sprintf(
```

Comment:
This is fine as it is explicitly referencing a global function rather than PHP having to look for a namespaced version, no?

Reply:
Yea, but it's unnecessary, no? It implies that there is a namespaced `sprintf`, well to me at least.

Reply:
It's not strictly necessary, no. I think it falls in the territory of a micro-optimization.

Reply:
It would actually be best practice to only ever use fully qualified identifiers (with imports as needed to avoid visual clutter), as that is best for performance and makes code comprehension for the IDE unambiguous.

In a case like this, if you don't use a fully qualified identifier (i.e. `\sprintf` or using an import), PHP would first check for the existence of `AmpProject\AmpWp\Instrumentation\sprintf` and after not having found it fall back to the global `sprintf`. Use only fully qualified identifiers avoids this first lookup.

I'm fine with removing these, but is this maybe something to discuss first?